samples/pet-store/cellery/resources/pet-store.swagger.json

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
Please remove this swagger file.
This is already present in https://github.com/wso2-cellery/sdk/blob/master/samples/pet-store/components/controller/resources/pet-store.swagger.json